### DISCUSSION TITLE
2015/04/21/salt-lake-city-to-boise-beacon-25

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#17] [Announcements]
# 2015/04/21/salt-lake-city-to-boise-beacon-25

http://localhost:4000/2015/04/21/salt-lake-city-to-boise-beacon-25

**Comment by jengalas** [comment]
Posted by Kaarin Engelmann
October 15, 2016 at 1:21 pm
_Imported from WordPress_

Hi! Are you still interested in the beacon system in Idaho? I’m trying to put together a project for the Idaho Historical Society on the system–working with the Idaho Air Association and others. It would be great to get more people on board. Where are you located? My email address is kengelm@yahoo.com.

**Reply by jengalas** [reply]
Posted by Glen Smallwood
October 27, 2016 at 4:46 pm
_Imported from WordPress_

Kaarin; still working on them. Located in Boise. Sent you an email a couple of weeks ago with contact info.
